src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,136 @@
+                                          ItemsSource="{Binding FontSizeList}"
+                                          SelectedItem="{Binding Path=SelectedFontSize}"
+                                          Style="{StaticResource NoBordersComboBoxStyle}">
+                                </ComboBox>
+                                
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Run Preview options-->
+                            <Grid Grid.Row="3">
+                                <Grid>
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto"/>
+                                        <ColumnDefinition Width="*"/>
+                                    </Grid.ColumnDefinitions>
+                                    <ToggleButton MinWidth="30"             
+                                                  MaxHeight="20"
+                                                  VerticalAlignment="Center"
+                                                  Grid.Column="0"                              
+                                                  IsChecked="{Binding Path=RunPreviewIsChecked}"
+                                                  Style="{StaticResource EllipseToggleButton1}"/>
+                                    <Label Grid.Column="1"
+                                           Content="Show Run Preview" 
+                                           VerticalAlignment="Center"
+                                           Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                </Grid>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Number Format options-->
+                            <Grid Grid.Row="4">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Label Grid.Row="0"
+                                       Content="Number Format" 
+                                       VerticalAlignment="Center"
+                                       Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                <ComboBox Name="numberFormatCmb"
+                                          Grid.Row="1"                                         
+                                          SelectedItem="{Binding Path=SelectedNumberFormat}"
+                                          ItemsSource="{Binding NumberFormatList}"
+                                          Style="{StaticResource NoBordersComboBoxStyle}"
+                                          Height="30"
+                                          Width="390">
```

Comment:
The width here seems inconsistent with other ComboBox?

Reply:
@QilongTang  I fixed the combobox width in the commit:
https://github.com/DynamoDS/Dynamo/pull/11542/commits/89b81d746df5a9d76bb3597c2f49446c392abae8